opentelemetry/proto/common/v1/common.proto

```diff
@@ -21,30 +21,37 @@ option java_package = "io.opentelemetry.proto.common.v1";
 option java_outer_classname = "CommonProto";
 option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/common/v1";
 
+// AnyValue is used to represent any type of attribute value. AnyValue may contain a
+// primitive value such as a string or integer or it may contain an arbitrary nested
+// object containing arrays, key-value lists and primitives.
+message AnyValue {
+  // The value is one of the listed fields. It is valid for all values to be unspecified
+  // in which case this AnyValue is considered to be "null".
+  oneof value {
+    string string_value = 1;
+    bool bool_value = 2;
+    int64 int_value = 3;
+    double double_value = 4;
+    ArrayValue array_values = 5;
+    AttributeKeyValueList kvlist_values = 6;
+  }
+}
+
+message ArrayValue {
+  // Array of values. The array may be empty (contain 0 elements).
+  repeated AnyValue array_values = 1;
```

Comment:
why is this message needed? can you have `repeated ArrayValue array_values = 5;` in the AnyValue?

Reply:
Same, required by oneof.

Reply:
Can we name this just "values"?  It's already inside a thing named "ArrayValue", so "values" seems like enough information.

Reply:
Renamed to `values`.